cli/lib/errors.js

```diff
@@ -9,13 +9,36 @@ const state = require('./tasks/state')
 
 const docsUrl = 'https://on.cypress.io'
 const requiredDependenciesUrl = `${docsUrl}/required-dependencies`
+const runDocumentationUrl = `${docsUrl}/command-line#cypress-run`
```

Comment:
```suggestion
const runDocumentationUrl = `${docsUrl}/cypress-run`
```

Reply:
Adding this redirect in https://github.com/cypress-io/cypress-services/pull/2714